### ISSUE TITLE
Remove/Fix Repo from project

### ISSUE THREAD

**Issue by yordis** [mustafaturan/authable#12]
Current I am using this configuration

```
config :authable,
  ecto_repos: [MyApp.Repos.OAuthRepo],
  repo: MyApp.Repos.OAuthRepo,
```

repo
```
defmodule MyApp.Repos.OAuthRepo do
  use Ecto.Repo, otp_app: :my_app
end
```

As you can see I create my own repo that I want to use as the `authable` repo.

But right now I am getting an error
```
Compilation error on file lib/authable/repo.ex ==
** (ArgumentError) missing :adapter configuration in config :authable, Authable.Repo
```

I guess the reason of this is that I didn't configure `Authable.Repo` which indeed I didn't because I used `MyApp.Repos.OAuthRepo`

How can I fix this without having a ghost config in my configuration?
Should we remove `Authable.Repo` and let the user to use the `Repo` that they want so this is not longer an issue?

**Comment by mustafaturan** [comment]
In that case, I will need to remove OTP app too, but give me a few weeks to consider it. During that time you can use experimental branch [`norepo`](https://github.com/mustafaturan/authable/tree/norepo) and share your experience with that branch.

**Comment by yordis** [comment]
@mustafaturan worked on

**Comment by kerryjj** [comment]
I'm seeing this issue when trying to use shield, is it because shield depends on v0.8.0 or authable? I also can't see a tag for v0.8.0 in this repo anymore.

Is this fixed in v0.9.1?
